docs/command/atlas-projects-update.txt

```diff
@@ -0,0 +1,101 @@
+.. _atlas-projects-update:
+
+=====================
+atlas projects update
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Update a project.
+
+To use this command, you must authenticate with a user account or an API key with the Project Data Access Read/Write role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas projects update <projectId> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - projectId
+     - string
+     - true
+     - ID of the project you want to update.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -f, --file
+     - string
+     - true
+     - Path to the JSON configuration file that defines project configuration settings. To learn more about project configuration files for the Atlas CLI, see https://dochub.mongodb.org/core/tbd!!!!!!!!!.
```

Comment:
todo: get dochub url

Reply:
Generated, I commented below to change the text to the new URL